server/src/resolvers/mutation.js

```diff
@@ -15,13 +12,13 @@ module.exports.Mutation = {
       };
     }
 
-    // TODO(lsh9034): Implement session logic.
-    // Create session and return sessionId to client.
+    // TODO(lsh9034): modify expires time depend on last user interaction.
+    const token = jwt.sign({userId:userId}, APP_SECRET,{expiresIn:'100h'});
```

Comment:
Token expiration seems to be set to different time duration between be and fe (or at least they're declared in multiple different places). Can we find ways to unify them?